src/PKSim.Assets/PKSimConstants.cs

```diff
@@ -2202,6 +2202,14 @@ public static class UI
          public static readonly string MoleculeExcludeList = "For All Molecules Except";
          public static readonly string MoleculeIncludeList = "For Molecules";
          public static readonly string Type = "Type";
+         public static readonly string LocalizationIntracellular = "Intracellular";
+         public static readonly string LocalizationInterstitial = "Interstitial";
+         public static readonly string LocalizationBloodCellsIntracellular = "Blood Cells Intracellular";
+         public static readonly string LocalizationBloodCellsMembrane = "Blood Cells Membrane";
+         public static readonly string LocalizationVascEndosome = "Vasc. Endothelium endosome";
+         public static readonly string LocalizationVascMembraneApical = "Vasc. Endothelium membrane plasma-side";
+         public static readonly string LocalizationVascMembraneBasolateral = "Vasc. Endothelium membrane tissue-side";
```

Comment:
Non-consistent: Blood cells written all in capital letters but VascEndo not:
So would write either "Blood Cells **m**embrane" etc. or "Vascular Endothelium **E**ndosome" etc.

Reply:
Agreed.. all copy paste from the ppt
I would in fact say it should be Blood cells membrane no?

Reply:
probably yes

Reply:
done